src/types/quiz.types.ts

```diff
@@ -0,0 +1,51 @@
+import {
+	ADD__ANSWER,
+	CLOSE__MODAL,
+	OPEN__MODAL,
+	SET__PLAYER__NAME,
+	SET__QUESTIONS,
+} from "../constants";
+
+export type Question = {
+	_id: string;
+	category: string;
+	correctAnswer: string;
+	options: string[];
+	question: string;
+	imageUrl?: string;
+};
+
+export type LoadedQuestions = {
+	questions: Question[];
+};
+
+export type ModalState = {
+	isModalOpen: boolean;
+	modalType?: string;
+	data?: string;
+};
+
+export type AnswerObject = {
+	question: string;
+	answer: string;
+	correct: boolean;
+	correctAnswer: string;
+};
+
+export type QuestionState = Question & { answers: string[] };
```

Comment:
I don't know whether this is a good way or not to declare `answers` as `string` array in `QuestionState`, 
this might work now but in the long run, you need objects to extend this application
You can ask this to our captain, maybe I am not able to clearly explain

Reply:
Actually, I didn't use this type anywhere in the code. Removed it!